packages/cubejs-duckdb-driver/src/DuckDBDriver.ts

```diff
@@ -55,10 +55,39 @@ export class DuckDBDriver extends BaseDriver implements DriverInterface {
     return super.toGenericType(columnType.toLowerCase());
   }
 
+  private async installExtensions(extensions: string[], execAsync: (sql: string, ...params: any[]) => Promise<void>, repository: string = ''): Promise<void> {
```

Comment:
Maybe it would be better to mark repository as optional (and as it is in reality):
```suggestion
  private async installExtensions(extensions: string[], execAsync: (sql: string, ...params: any[]) => Promise<void>, repository?: string): Promise<void> {
```